integration-test/langfuse-integration-error-handling.spec.ts

```diff
@@ -0,0 +1,200 @@
+// uses the compiled fetch version, run yarn build after making changes to the SDKs
+import Langfuse from "../langfuse";
+import { CallbackHandler } from "../langfuse-langchain";
+
+import { FakeListLLM } from "langchain1/llms/fake";
+
+import { LF_HOST, LF_PUBLIC_KEY, LF_SECRET_KEY } from "./integration-utils";
+
+describe("No errors should be thrown by SDKs", () => {
+  jest.useRealTimers();
+
+  // beforeEach(() => {});
+  // afterEach(async () => {});
+
+  describe("langfuse-fetch", () => {
+    it("incorrect host", async () => {
+      global.console.error = jest.fn();
+      const langfuse = new Langfuse({
+        publicKey: LF_PUBLIC_KEY,
+        secretKey: LF_SECRET_KEY,
+        baseUrl: "https://incorrect-host",
+        flushAt: 2,
+        fetchRetryDelay: 1,
+        fetchRetryCount: 2,
+      });
+
+      const trace = langfuse.trace({ name: "trace-name" });
+      for (let i = 0; i < 10; i++) {
+        trace.generation({ name: "generation-name" });
+      }
+
+      await new Promise((resolve) => setTimeout(resolve, 4000));
+      await langfuse.shutdownAsync();
+
+      // expect no errors to be thrown (would kill jest) and console.error to be called
+      expect(true).toBe(true);
```

Comment:
expect(true).toBe(true); is not necessary here i think

Reply:
yes, initially only had this test, now also have another one. removing this.

Reply:
agree, removed